include/openssl/rand.h

```diff
@@ -104,6 +104,9 @@ OPENSSL_EXPORT OPENSSL_DEPRECATED void RAND_add(const void *buf, int num,
 // RAND_egd returns 255.
 OPENSSL_EXPORT OPENSSL_DEPRECATED int RAND_egd(const char *);
 
+// RAND_egd_bytes returns 255.
+OPENSSL_EXPORT OPENSSL_DEPRECATED int RAND_egd_bytes(const char *);
```

Comment:
wrong signature?

Reply:
yeah... my bad, got caught copying this from `RAND_egd`. Also updated documentation along with this.